digdag-standards/src/main/java/io/digdag/standards/operator/gcp/BqClient.java

```diff
@@ -37,7 +40,10 @@
     protected Bigquery client(GoogleCredential credential, HttpTransport transport, JsonFactory jsonFactory)
     {
         if (credential.createScopedRequired()) {
-            credential = credential.createScoped(BigqueryScopes.all());
+            Set<String> scopes = new HashSet<String>();
+            scopes.addAll(BigqueryScopes.all());
+            scopes.add(SheetsScopes.DRIVE_READONLY);
```

Comment:
I am not sure google credential but I have a concern about adding scope affect to existence users or not.
It means that users need to change their credential in secrets or change permission on account even though they don't need to access to Google spreadsheet.
Could you explain about it ?